components/odh-notebook-controller/config/webhook/kustomization.yaml

```diff
@@ -1,10 +1,9 @@
----
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - manifests.yaml
-  - service.yaml
+- manifests.yaml
+- service.yaml
 commonAnnotations:
   service.beta.openshift.io/inject-cabundle: "true"
 configurations:
-  - kustomizeconfig.yaml
+- kustomizeconfig.yaml
```

Comment:
this indentation change would fail on notebook repository as we have the yaml indentation check in the CI there... but let's not bother with this here now since it's automatically generated